x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/index.test.ts

```diff
@@ -149,7 +193,18 @@ describe('getTransactionBreakdown', () => {
           get: () => 'myIndex' as any,
           has: () => true
         },
-        uiFiltersES: []
+        uiFiltersES: [],
+        indices: {
+          apm_oss: {
```

Comment:
Felt a sudden urge to use the cool Proxy feature. But not should if it's better:
```suggestion
          apm_oss: new Proxy({}, { get() { return 'myIndex' } })
```